plugins/inputs/cisco_telemetry_mdt/cisco_telemetry_mdt.go

```diff
@@ -416,6 +416,10 @@ func (c *CiscoTelemetryMDT) handleTelemetry(data []byte) {
 		if content != nil {
 			// Parse values
 			for _, subfield := range content.Fields {
+				if len(subfield.Fields) == 0 {
+					continue
+				}
+
```

Comment:
I will quote @srebhan from https://github.com/influxdata/telegraf/pull/12637 ;)
"Hmmm, are you sure that `parseContentField` will always fail if there are not fields or does it in some cases return a sensible value?"

Reply:
haha good point - let's see what the user finds with respect to the crash first